### PR TITLE
Fixed request output to include url query.

### PR DESCRIPTION
**Pull request by jcru**
Initially output for pagination request:

GET /v1.0/1234/instances HTTP/1.1
User-Agent: python-example-client
Host: ord.databases.api.rackspacecloud.com
X-Auth-Token: 025e7b2add104c23a08903f89b8824fe
Accept: application/json
Content-Type: application/json

With the fix (includes query for first line url):
GET /v1.0/1234/instances?limit=1 HTTP/1.1
User-Agent: python-example-client
Host: ord.databases.api.rackspacecloud.com
X-Auth-Token: 55d741844e7c4979b4e944253aa2c4a2
Accept: application/json
Content-Type: application/json

### CODE REVIEW

**Comment by jcru** [comment]
I also renamed the pagination output files.

**Comment by TimSimpsonR** [comment]
+1

File: tests/examples/examples/example_generation.py [hunk 1]
```diff
@@ -163,7 +163,11 @@ def _indent_xml(self, my_string):
     def output_request(self, url, output_headers, body, content_type, method):
         output_list = []
         parsed = urlparse(url)
-        output_list.append("%s %s HTTP/1.1" % (method, parsed.path))
+        if parsed.query:
```
Comment (ed-) [inline tests/examples/examples/example_generation.py]: I **know** I'd added this to the example gen in the past; I have no idea how it didn't make it across in the trip to lite.
Glad you've saved it and added it again.

**Comment by ed-** [comment]
+1

**Comment by hub-cap** [comment]
victory +1

**Comment by rnirmal** [comment]
+1

File: tests/examples/examples/example_generation.py [hunk 2]
```diff
@@ -160,13 +160,23 @@ def _indent_xml(self, my_string):
...
         return my_string
 
-    def output_request(self, url, output_headers, body, content_type, method):
+    def output_request(self, url, output_headers, body, content_type, method,
+                       static_auth_token=True):
         output_list = []
         parsed = urlparse(url)
-        output_list.append("%s %s HTTP/1.1" % (method, parsed.path))
+        if parsed.query:
+            method_url = parsed.path + '?' + parsed.query
```
Comment (cp16net) [inline tests/examples/examples/example_generation.py:168]: good catch! :)

**Comment by cp16net** [comment]
you rock +1

**Pull request merged**